config/cfme/manifest_5_11.json

```diff
@@ -1,6 +1,7 @@
 {
   "cfme_version": "5.11",
   "manifest": {
+    "version": "1.0.0",
```

Comment:
This is going to require a change to @abellotti 's rejection code in the API - https://github.com/RedHatCloudForms/cfme-cloud_services/pull/23#discussion_r305049853

Reply:
@Fryguy https://github.com/RedHatCloudForms/cfme-cloud_services/pull/27